### PR TITLE
W-12413431-PlatformUpdateRN-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues for MuleSoft Composer] for a list of known issues by date.
 
+== February 8, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+3]
+
+[%header%autowidth.spread]
+|===
+|Issue Resolution |ID
+|Previously, when opening a flow, all steps of the flow were open by default. Now, when you open a flow, all steps aside from If/Else Blocks and For Each Loops are collapsed to provide a navigable outline of the flow |W-12515426
```
Comment (valkyrie69) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:17]: Was this really a bug? It seems like an enhancement. 

IMO, you can leave the first line out entirely. Your call though. 

Now, when you open a flow, all steps except **If/Else Blocks** and **For Each Loops**, are collapsed to provide a navigable outline of the flow.

**Review by valkyrie69 — APPROVED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc (hunk 1 above)
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:17]:
```suggestion
|Now, when you open a flow, all steps aside from *If/Else Blocks* and *For Each Loops*, are collapsed to provide a navigable outline of the flow. |W-12515426
```

File: en/salesforce/src/docs/ms_composer_release_notes_2023.adoc [hunk 2]
```diff
@@ -7,6 +7,16 @@ New versions are automatically available once they are released; you don't need
...
 See xref:ms_composer_release_notes_ki.adoc[Known Issues for MuleSoft Composer for Salesforce] for a list of known issues by date.
 
+== February 8, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+3]
+
+[%header%autowidth.spread]
+|===
+|Issue Resolution |ID
+|Previously, when opening a flow, all steps of the flow were open by default. Now, when you open a flow, all steps aside from If/Else Blocks and For Each Loops are collapsed to provide a navigable outline of the flow |W-12515426
```
Comment (emileeburkeVT) [inline en/salesforce/src/docs/ms_composer_release_notes_2023.adoc:17]:
```suggestion
|Now, when you open a flow, all steps aside from *If/Else Blocks* and *For Each Loops*, are collapsed to provide a navigable outline of the flow. |W-12515426
 |W-12515426
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues for MuleSoft Composer] for a list of known issues by date.
 
+== February 8, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+3]
+
+[%header%autowidth.spread]
+|===
+|Issue Resolution |ID
+|Now, when you open a flow, all steps aside from *If/Else Blocks* and *For Each Loops*, are collapsed to provide a navigable outline of the flow. |W-12515426
```
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:17]:
```suggestion
|Now, when you open a flow, all steps aside from *If/Else Blocks* and *For Each Loops* are collapsed to provide a navigable outline of the flow. |W-12515426
```

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**